code/__HELPERS/game.dm

```diff
@@ -7,6 +7,8 @@
 
 #define Z_TURFS(ZLEVEL) block(locate(1,1,ZLEVEL), locate(world.maxx, world.maxy, ZLEVEL))
 
+#define HOLIDAY_ACTIVE(holiday)  SSevent.holidays && SSevent.holidays[##holiday]

```

Comment:
I think the holiday events don't exist in the list if they're not there, you might need to add an explicit

if x in thing

to prevent the runtime errors.

or change how the holiday events list works to have a true in valid event indexes and false in the rest (which to me seems saner)